templates/careers/application/partial/_assessment.html

```diff
@@ -1,38 +1,83 @@
+  <h3 class="p-heading--2 p-process-stepped-list__title--stepper-{{ assessment_stage }}">Assessment</h3>
   <div class="p-stepped-list__content">
-    {% if "stage_progress" in application and application["stage_progress"]["assessment"] and application["stage_progress"]["early_stage"] %}
+    {% if is_early_stage %}
       <p>There are several parts to this stage:</p>
-      <a href="" onclick="showProgressDetail('assessment'); event.preventDefault();" class="show-more-assessment">Show more</a>
-      <div class="progress-detail-assessment u-hide">
-    {% else %}
-      <div class="progress-detail-assessment">
+      <a href=""
+         onclick="showProgressDetail('assessment'); event.preventDefault();"
+         class="show-more-assessment">Show more</a>
     {% endif %}
-        <ol class="p-stepped-list">
-          <li class="p-stepped-list__item">
-            <p class="p-stepped-list__title u-no-margin--bottom">The Written Interview: As the first part of this stage we invite you to submit your answers to some questions specifically designed so that we can find out more about your experience and skill set. This is your opportunity to highlight your key strengths for this position. To ensure that we mitigate against unconscious bias in our hiring process we ask that you refrain from including any personal identifiable information in this document.</p>
-          </li>
-          <li class="p-stepped-list__item">
-            <p class="p-stepped-list__title">Psychometric assessment: As part of our professional review this assessment will help us to understand your fluid intelligence.</p>
-            {% if application["gia_feedback"] %}
-            <p style="padding-left: 2.5rem; grid-column-end: span 12;" class="u-no-margin--bottom" id="reportRequestContainer">
-              <button id="showRequestAssessmentModal" aria-controls="request-assessment-modal">Request assessment report</button>
+    <div class="progress-detail-assessment {{ 'u-hide' if is_early_stage }}">
+      <ol class="p-stepped-list">
+        <li class="p-stepped-list__item">
+          <p class="p-stepped-list__title">
+            The Written Interview: As the first part of this stage we invite you to submit your answers to some questions specifically designed so that we can find out more about your experience and skill set. This is your opportunity to highlight your key strengths for this position. To ensure that we mitigate against unconscious bias in our hiring process we ask that you refrain from including any personal identifiable information in this document.
+          </p>
+          {% if application["custom_fields"]["written_interview_submitted_at"] %}
+            <p style="padding-left: 2.5rem;
+                      grid-column-end: span 12"
```

Comment:
`grid-column-end: span 12` doesn't seem to have an effect

Reply:
Actually it does have some effect on a narrow screen. The styling was copypasted from a sibling <li>.

[Screencast from 2024-04-29 18-49-05.webm](https://github.com/canonical/canonical.com/assets/2834658/cea15e59-f183-48fe-9117-a41d3d3a3853)

Reply:
Alright, let's leave it then :+1: